src/Iterator/CsvJsonFileIterator.php

```diff
@@ -42,14 +42,10 @@ class CsvJsonFileIterator implements \Iterator
      */
     public function __construct($file)
     {
-        if (is_resource($file)) {
-            $this->handle = $file;
-        } else {
-            $this->handle = @ fopen($file, 'rb');
-
-            if ($this->handle === false) {
-                throw new \InvalidArgumentException('Cannot open file for reading: ' . $file);
-            }
+        $this->handle = (is_resource($file) ? $file : @ fopen($file, 'rb'));
+
+        if ($this->handle === false) {
+            throw new \InvalidArgumentException('Cannot open file for reading: ' . $file);
```

Comment:
Same here.

Reply:
What do you mean about this comment?

Reply:
> I don't like the null coalesce operator here, as it also allows for undefined variable without notice. This operator is equivalent to isset($values) ? $values : [], not strictly equivalent to the original code.

Reply:
This condition is not related to the null coalesce operator.

I think this change is proper approach.